bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VolvoOnCallBridgeHandler.java

```diff
@@ -67,9 +67,11 @@ public VolvoOnCallBridgeHandler(Bridge bridge) {
         super(bridge);
 
         httpHeader.put("cache-control", "no-cache");
-        httpHeader.put("X-OS-Type", "Android");
-        httpHeader.put("X-Originator-Type", "App");
-        httpHeader.put("Content-Type", JSON_CONTENT_TYPE);
+        httpHeader.put("content-type", JSON_CONTENT_TYPE);
+        httpHeader.put("x-device-id", "Device");
+        httpHeader.put("x-originator-type", "App");
+        httpHeader.put("x-os-type", "Android");
+        httpHeader.put("x-os-version", "22");
```

Comment:
Why these changes ?

Reply:
Not sure if required, although, included/corrected based on Volvo On Call bundle created for Athome Homey.

Reply:
Can you try without these changes. I remind that I reduced it as much as possible in earlier versions of the binding.

Reply:
Have tried with and without the additions. Preclimatization works with both, however, lock/unlock seems to work only with the additions (for me at least).

Reply:
Have tried both with and without additions. Preclimatization works for both, however, lock function seem to work only with the additions (at least for me).